vmdk_plugin/utils/test_util/VolumeCreationTestUtil.go

```diff
@@ -49,3 +52,83 @@ func DeleteVolume(name string, ip string) ([]byte, error) {
 func InvokeCommand(remoteHostIP string, cmd string) ([]byte, error) {
 	return exec.Command("/usr/bin/ssh", append(SSH_OPTS, "root@"+remoteHostIP, cmd)...).CombinedOutput()
 }
+
+// returns disk format of volume using Admin cli
+func GetDiskFormatUsingAdminCli(volName string, hostname string) string {
```

Comment:
@msterin Thanks for the review. 
Yes, we can get all the needed information in one shot but I was thinking if we use this approach, we will come to know of any issues only at the end.

For e.g., we created a volume of capacity 100MB and admin cli is reporting incorrect value, then we can fail the test there itself. If we see capacity is correct, we do not need to run the container or do any more steps.